### PR TITLE
[PERFSCALE-3494] Adds Large Scale CP Tests

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds CP Tests to OCP 4.18 Large Scale Cluster

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-control-plane-252nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml [hunk 1]
```diff
@@ -36,6 +36,25 @@ tests:
     - chain: openshift-qe-orion-consolidated
     workflow: openshift-qe-installer-aws
   timeout: 3h30m0s
+- as: control-plane-252nodes
+  cron: 0 2 11,25 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:40]: use: `0 0 25 * *` for once/month cadence.

**Comment by qiliRedHat** [comment]
@krishvoor ci/prow/ci-operator-registry job failed, otherwise looks good to me.

**Pull request merged**